02_configure_host.sh

```diff
@@ -196,7 +196,7 @@ if [[ "$reg_state" == "exited" ]]; then
   sudo "${CONTAINER_RUNTIME}" start registry
 elif [[ "$reg_state" != "running" ]]; then
   sudo "${CONTAINER_RUNTIME}" rm registry -f || true
-  sudo "${CONTAINER_RUNTIME}" run -d -p "${REGISTRY_PORT}":5000 --name registry "$DOCKER_REGISTRY_IMAGE"
+  sudo "${CONTAINER_RUNTIME}" run -d -p "${REGISTRY}":5000 --name registry "$DOCKER_REGISTRY_IMAGE"
```

Comment:
I think we should drop the port, right?
```suggestion
  sudo "${CONTAINER_RUNTIME}" run -d -p "${REGISTRY}" --name registry "$DOCKER_REGISTRY_IMAGE"
```

Reply:
No, the syntax is `IP:LOCAL PORT:CONTAINER PORT`.

Reply:
Thanks for explaining!